README.md

```diff
@@ -427,13 +421,15 @@ curl --request GET 'http://localhost:8080/service-apis/v1/allServiceAPIs?api-inv
 [Test Plan Directory](./docs/test_plan/README.md)
 
 
-Additional info:
+
+# Important urls:
+
+## Mongo DB Dashboard
 ```
-post_body_example.json:
-Example of APIInvokerEnrolmentDetails object for POST request of API_Invoker_Management API 
+http://0.0.0.0:8081/ (if accessed from localhost) 
 
+or
 
-Mongo Express url:
-http://0.0.0.0:8081/
+http://<Mongo Express Host IP>:8081/ (if accessed from another host)
```

Comment:
According to change at docker-compose, port exposed by mongo-express will be 8082